src/openms/source/ANALYSIS/XLMS/OPXLSpectrumProcessingAlgorithms.cpp

```diff
@@ -137,7 +140,7 @@ namespace OpenMS
 
       if (deisotope_spectra)
       {
-        PeakSpectrum deisotoped = OPXLSpectrumProcessingAlgorithms::deisotopeAndSingleChargeMSSpectrum(exp[exp_index], 1, 7, fragment_mass_tolerance_xlinks, fragment_mass_tolerance_unit_ppm, false, 3, 10, false);
+        PeakSpectrum deisotoped = OPXLSpectrumProcessingAlgorithms::deisotopeAndSingleChargeMSSpectrum(exp[exp_index], 1, 7, 100, fragment_mass_tolerance_unit_ppm, false, 3, 10, false);
```

Comment:
fix this to 100ppm?

Reply:
oh right, I think I should change that. Do you think using the search tolerance for deisotoping as well is the best way to go? And I think I will also include a parameter into the tools to let the user decide, if they want to apply deisotoping. Currently I only deisotope the spectra if the tolerance is below 100 ppm, but that was just a sloppy way to separate the two types of data I have seen so far: > 0.1 Da tolerances from ion traps and < 30 ppm tol from orbitraps. Maybe I will leave that as an "auto" mode, but give the user the ability to decide as an advanced parameter.